packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -391,7 +413,44 @@ class _MethodStreamManager {
     WebSocket webSocket,
     Session session,
     SerializationManager serializationManager,
+    _RequiredAuthentication? requiredAuthentication,
   ) {
+    late void Function(SerializableModel) authenticationRevokedCallback;
+
+    if (requiredAuthentication != null) {
+      // If authentication was required, a listener is registered that closes
+      // the stream if the authentication token is revoked.
```

Comment:
Question: Not sure what "comment policy" we have in this project. But generally for code such as this (which is very clean, short and clear) I wouldn't consider it necessary.

Reply:
Agree, comments have a tendency to become outdates so we should limit them if possible.
My fear here is this class is a bit of a brainchild of mine and I want to leave as many clues as possible for the next engineer to figure out what is happening. But this should only be limited to required comments so will remove this one.

Reply:
Thanks for the context, makes sense!